seqseek/lib.py

```diff
@@ -8,75 +8,143 @@
-    '11': 135006516,
-    '12': 133851895,
-    '13': 115169878,
-    '14': 107349540,
-    '15': 102531392,
-    '16': 90354753,
-    '17': 81195210,
-    '18': 78077248,
-    '19': 59128983,
-    '20': 63025520,
-    '21': 48129895,
-    '22': 51304566,
-    'X':  155270560,
-    'Y':  59373566,
-    'MT': 16571,
 
-    # Haplotype contigs
-    '6_apd_hap1': 4622290,
-    '6_cox_hap2': 4795371,
-    '6_dbb_hap3': 4610396,
-    '6_mann_hap4': 4683263,
-    '6_mcf_hap5': 4833398,
-    '6_qbl_hap6': 4611984,
-    '6_ssto_hap7': 4928567,
-    '17_ctg5_hap1': 1680828,
+BUILD37_ACCESSIONS = {
+    '1': 'NC_000001.10',
+    '2': 'NC_000002.11',
+    '3': 'NC_000003.11',
+    '4': 'NC_000004.11',
+    '5': 'NC_000005.9',
+    '6': 'NC_000006.11',
+    '7': 'NC_000007.13',
+    '8': 'NC_000008.10',
+    '9': 'NC_000009.11',
+    '10': 'NC_000010.10',
+    '11': 'NC_000011.9',
+    '12': 'NC_000012.11',
+    '13': 'NC_000013.10',
+    '14': 'NC_000014.8',
+    '15': 'NC_000015.9',
+    '16': 'NC_000016.9',
+    '17': 'NC_000017.10',
+    '18': 'NC_000018.9',
+    '19': 'NC_000019.9',
+    '20': 'NC_000020.10',
+    '21': 'NC_000021.8',
+    '22': 'NC_000022.10',
+    'X': 'NC_000023.10',
+    'Y': 'NC_000024.9',
+    'MT': 'NC_012920.1',
+
+    # UCSC names for haplotype contigs
+    'chr6_apd_hap1': 'NT_167244.1',
+    'chr6_cox_hap2': 'NT_113891.2',
+    'chr6_dbb_hap3': 'NT_167245.1',
+    'chr6_mann_hap4': 'NT_167246.1',
+    'chr6_mcf_hap5': 'NT_167247.1',
+    'chr6_qbl_hap6': 'NT_167248.1',
+    'chr6_ssto_hap7': 'NT_167249.1',
+    'chr4_ctg9_hap1': 'NT_167250.1',
+    'chr17_ctg5_hap1': 'NT_167251.1'
 }
 
-# chromosome names and lengths for build 38
-BUILD38_CHROMOSOMES = {
-    '1':	248956422,
-    '2':	242193529,
-    '3':	198295559,
-    '4':	190214555,
-    '5':	181538259,
-    '6':	170805979,
-    '7':	159345973,
-    '8':	145138636,
-    '9':	138394717,
-    '10':	133797422,
-    '11':	135086622,
-    '12':	133275309,
-    '13':	114364328,
-    '14':	107043718,
-    '15':	101991189,
-    '16':	90338345,
-    '17':	83257441,
-    '18':	80373285,
-    '19':	58617616,
-    '20':	64444167,
-    '21':	46709983,
-    '22':	50818468,
-    'X':	156040895,
-    'Y':	57227415,
-    'MT':   16569
+BUILD38_ACCESSIONS = {
+    '1': 'NC_000001.11',
+    '2': 'NC_000002.12',
+    '3': 'NC_000003.12',
+    '4': 'NC_000004.12',
+    '5': 'NC_000005.10',
+    '6': 'NC_000006.12',
+    '7': 'NC_000007.14',
+    '8': 'NC_000008.11',
+    '9': 'NC_000009.12',
+    '10': 'NC_000010.11',
+    '11': 'NC_000011.10',
+    '12': 'NC_000012.12',
+    '13': 'NC_000013.11',
+    '14': 'NC_000014.9',
+    '15': 'NC_000015.10',
+    '16': 'NC_000016.10',
+    '17': 'NC_000017.11',
+    '18': 'NC_000018.10',
+    '19': 'NC_000019.10',
+    '20': 'NC_000020.11',
+    '21': 'NC_000021.9',
+    '22': 'NC_000022.11',
+    'X': 'NC_000023.11',
+    'Y': 'NC_000024.10',
+    'MT': 'NC_012920.1',
+}
+
+# chromosome names and lengths for build 37
+ACCESSION_LENGTHS = {
```

Comment:
Can these serve as a validation of the downloaded file to make sure nothing weird happened?